server/models/user.go

```diff
@@ -1,14 +1,20 @@
 package models
 
 import (
+	"time"
+
 	"gorm.io/gorm"
 )
 
 type User struct {
 	gorm.Model
-	Email    string `json:"email"`
-	Password string `json:"password"`
-	FullName string `json:"fullName" gorm:"column: full_name"`
+	Email    	string 	  `json:"email"`
+	Password    string    `json:"password"`
+	FullName    string    `json:"fullName" gorm:"column: full_name"`
+	IsSuperuser bool      `json:"isSuperuser" gorm:"is_superuser;default:false"`
+	CreatedAt   time.Time `gorm:"autoCreateTime: true"`
+	UpdatedAt   time.Time `gorm:"autoUpdateTime: true"`
+	
```

Comment:
```suggestion
```
Remove those, since we are using `gorm.Model` in the struct it will be created automatically